src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -122,7 +122,7 @@ public static JsonObject of(String k1, Object v1) {
    * @return a JsonObject containing the specified mappings.
    */
   public static JsonObject of(String k1, Object v1, String k2, Object v2) {
-    JsonObject obj = new JsonObject(new LinkedHashMap<>(2));
+    JsonObject obj = new JsonObject(new LinkedHashMap<>(4));
```

Comment:
```suggestion
    JsonObject obj = new JsonObject(new LinkedHashMap<>(3));
```

According to https://guava.dev/releases/21.0/api/docs/src-html/com/google/common/collect/Maps.html#line.291